contracts/MplRewards.sol

```diff
@@ -145,27 +148,35 @@ contract MplRewards is Ownable {
         emit RewardAdded(reward);
     }
 
-    // End rewards emission earlier
+    /**
+        @dev End rewards emission earlier. Only the contract Owner may call this.
+    */
     function updatePeriodFinish(uint timestamp) external onlyOwner {
         _updateReward(address(0));
         periodFinish = timestamp;
     }
 
-    // Added to support recovering LP Rewards from other systems such as BAL to be distributed to holders
+    /**
+        @dev Added to support recovering LP Rewards from other systems such as BAL to be distributed to holders.
```

Comment:
Lets change this to 
`Added to support recovering tokens sent into this contract by users unintentionally.`